ecosystem/sep-0006.md

```diff
@@ -75,7 +75,51 @@ This protocol involves the transfer of value, and so HTTPS is required for all e
 
 This protocol was originally designed to provide conversions between off-chain and on-chain equivalent assets – for instance `BRL <> BRLT`, `USD <> USDC`, `NGN <> NGNT`, etc. – but it also describes how to make conversions between off-chain and on-chain non-equivalent assets like `BRL <> USDC`.
 
-To enable the conversion of non-equivalent assets the Anchor needs to: (i) mark the asset with `"quotes_supported": true` in its [`/info`](#info) endpoint; and (ii) implement the [SEP-38 Anchor RFQ API](sep-0038.md), which enables the Anchor to provide quotes to the Wallet. These quotes can be [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) or [firm](https://www.investopedia.com/terms/f/firmquote.asp). If a firm quote is used, the Stellar transaction _must_ be submitted to the Stellar network prior to the quote's expiration. More specifically, the `created_at` timestamp returned in the [Horizon GET /transactions/:id](https://developers.stellar.org/api/resources/transactions/) request must be earlier than the quote's expiration.
+To enable the conversion of non-equivalent assets the Anchor needs to implement the [SEP-38 Anchor RFQ API](sep-0038.md), which enables the Anchor to provide quotes to the Wallet. These quotes can be [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) or [firm](https://www.investopedia.com/terms/f/firmquote.asp).
+
+### Asset Conversions With Indicative Quotes
+
+For indicative quotes, users won't have a firm guarantee of the asset price they are going to pay. Here is how the conversion should happen in deposits and withdrawals for an asset pair like `BRL <> USDC`:
+
+**Deposit:**
+
+Considering the user will be making a bank deposit of fiat `BRL` in exchange for `USDC` in the Stellar network:
+
+1. The Wallet checks the assets' conversion rate in [`SEP-38 GET /price`](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0038.md#get-price) and shows the expected price to the user.
```

Comment:
I would change the word "expected" to "estimated", and maybe add a note about how the rate returned in the price at that moment and can change.

Reply:
Good suggestion! Updated in https://github.com/stellar/stellar-protocol/pull/973/commits/e25f8b306d471b13862b7d62f94a82773899dd3e.